src/util/normalizeConfig.js

```diff
@@ -193,17 +222,19 @@ export function normalizeConfig(config) {
       return flagEnabled(config, 'relativeContentPathsByDefault')
     })(),
 
-    files: (() => {
-      let { content, purge } = config
+    files: auto
+      ? 'auto'
+      : (() => {
+          let { content, purge } = config
```

Comment:
if content and purge is undefined returning `[]` might produce a fast path for this function.

Reply:
Good catch! Won't really matter from a performance perspective since this is only called once but added it either way. Thanks!